modules/notes.rb

```diff
@@ -32,12 +32,25 @@ def self.parse_args_addnote(text)
   end
   # rubocop: enable Metrics/MethodLength
 
+  def self.has_addnote_access?(event)
+    required_role_ids = Array(QBot.config&.roles&.addnote_access)
```

Comment:
Again, this should be a per-server configuration option. Make sure to specify in the option's description that an empty list means that everyone has addnote permissions.

Option names and descriptions are kept in the locale definitions: https://github.com/arch-community/qbot/blob/main/share/locales/en.yml